styles.css

```diff
@@ -0,0 +1,83 @@
+img {
+  max-width: 100%;
+}
+
+.card {
+  position: relative;
+  margin-right: auto;
+  margin-left: auto;
+  top: 200px;
+  width: 580px;
+}
+
+.action-list {
+  display: none;
+  list-style: none;
+  margin: 0;
+  padding-left: 0;
+  position: absolute;
+  bottom: 20px;
+  text-align: center;
+  letter-spacing: 60px;
+  width: 100%;
+}
+
+.action-list li {
+  display: inline-block;
+}
+
+.action-list img {
+  height: auto;
+  width: 50px;
+}
+
+.content {
+  color: white;
+  display: none;
+  font-family: "Roboto", sans-serif;
+  padding: 24px;
+  position: absolute;
+  bottom: 0;
+}
+
+.content p {
+  font-size: 20.5px;
+}
+
+.info {
+  font-size: 20.5px;
+  list-style: none;
+  line-height: 1.4;
+  margin: 0;
+  padding-left: 0;
+  text-align: right;
+  position: relative;
+  top: 20px;
+}
+
+.rating {
+  height: auto;
+  position: relative;
+  left: 14px;
+  width: 50px;
+}
+
+.card:hover img {
+  opacity: 0.8;
+}
+
+.card:hover .action-list {
+  display: block;
+}
+
+.card.is-displaying-content .content {
+  display: block;
+}
+
+.card.is-displaying-content .action-list {
+  display: none;
+}
+
+body {
+  background-color: black;
+}
```

Comment:
The `body` element selector should always be somewhere close to the top of your stylesheet, since it sets the ground rules for everything else.